based.py

```diff
@@ -245,6 +266,12 @@ def statement(self, p):
     @_("if_statement")
     def statement(self, p):
         return f"{p.if_statement}"
+    @_("return_statement")
```

Comment:
Put the "END" here rather than at return_statement in line 344, so just that its consistent